docs/administration/configuration/configuring-interactive-sessions/index.md

```diff
@@ -0,0 +1,53 @@
+# Configuring interactive sessions
+
+## Interactive sessions cleanup
+
+### Automatic and scheduled closure
+
+By default, interactive sessions in REANA are kept open indefinitely. However, administrators
+have the flexibility to configure the maximum inactivity period after which interactive
+sessions will be automatically closed; the schedule of this cleanup process can be
+customized. This helps optimize resource utilization and ensure that unused sessions
+are not consuming unnecessary resources.
+
+The automatic closure of interactive sessions in REANA can be customized by configuring
+[Helm values](https://github.com/reanahub/reana/tree/master/helm/reana).
+The following Helm values allow you to control the behavior of the automatic closure feature:
+
+- `interactive_sessions.maximum_inactivity_period`:
+  This option allows you to set a limit (in days) for the maximum inactivity period of interactive sessions after which they will be closed.
+
+- `interactive_sessions.cronjob_schedule`:
+  This option allows you to set the schedule for the cleanup process using the cronjob format.
+
+Example:
+
+```yaml
+interactive_sessions:
+    maximum_inactivity_period: 7
+    cronjob_schedule: "0 3 * * *"
+...
+```
+
+In the above example, interactive sessions that have been inactive for more than 7 days
+will be automatically closed, and the cleanup check will be performed every day at 3 AM.
+
+### Manual cleanup
+
+In addition to the automatic closure and scheduled cleanup, administrators can also
+manually clean up interactive sessions that have been inactive for a certain number of days.
+
+To manually clean up interactive sessions, use the `interactive-session-cleanup` command
+of the `reana-admin` tool that is available from the `reana-server` pod.
```

Comment:
```suggestion
This can be achieved with the `interactive-session-cleanup` command
of the `reana-admin` tool.
```

What do you think about moving this to the previous paragraph, while also making it shorter?